core/variant/variant.cpp

```diff
@@ -1133,82 +1071,102 @@ void Variant::reference(const Variant &p_variant) {
 
 	switch (p_variant.type) {
 		case NIL: {
-			// none
+			// None.
 		} break;
 
-		// atomic types
+		// Atomic types.
 		case BOOL: {
 			_data._bool = p_variant._data._bool;
+
```

Comment:
That goes against our current guidelines, I wouldn't add empty space that's not needed.

Reply:
Perfect, thank you. Addressed.